### DISCUSSION TITLE
Multiple compose files

### DISCUSSION THREAD

**Discussion by pascal-schrage** [microsoft/vscode-docker#2882] [Q&A] [answered]
Hello there, 

my current project contains multiple docker compose files in the development. Is there way to get the extension to execute a command with multiple compose files?
E.g.: `docker-compose -f ./docker-compose.yml -f ./docker-compose.dev.yml up -d`

Kind regards, 
Pascal

**Comment by bwateratmsft** [comment, marked as answer]
Yes, there are two different ways to do this!

The first is with [command customization](https://code.visualstudio.com/docs/containers/reference#_command-customization), where you can customize the command that gets assembled when doing the "Docker: Compose Up" action. It would probably make the most sense to put these command settings in the workspace rather than as global user settings.

The second is using a [`docker-compose` task](https://code.visualstudio.com/docs/containers/reference#_docker-compose-task) which I personally think works better for multiple files, but it's up to you.

**Reply by pascal-schrage** [reply]
Thank you very much. That did the trick for me. :)
